src/main/kotlin/kotlinx/kover/engines/intellij/IntellijVerification.kt

```diff
@@ -28,19 +27,19 @@ internal fun Task.intellijVerification(
     val aggRequest = File(temporaryDir, "agg-request.json")
     val groupedRules = groupRules(classFilter, rules)
     aggRequest.writeAggJson(projectFiles, groupedRules)
-    exec.javaexec { e ->
-        e.mainClass.set("com.intellij.rt.coverage.aggregate.Main")
-        e.classpath = classpath
-        e.args = mutableListOf(aggRequest.canonicalPath)
+    exec.javaexec {
+        mainClass.set("com.intellij.rt.coverage.aggregate.Main")
+        this@javaexec.classpath = classpath
+        args = mutableListOf(aggRequest.canonicalPath)
     }
 
     val verifyRequest = File(temporaryDir, "verify-request.json")
     val verifyResponseFile = File(temporaryDir, "verify-result.json")
     verifyRequest.writeVerifyJson(groupedRules, verifyResponseFile)
-    exec.javaexec { e ->
-        e.mainClass.set("com.intellij.rt.coverage.verify.Main")
-        e.classpath = classpath
-        e.args = mutableListOf(verifyRequest.canonicalPath)
+    exec.javaexec {
+        mainClass.set("com.intellij.rt.coverage.verify.Main")
+        this@javaexec.classpath = classpath
```

Comment:
`kotlin-dsl` plugin makes the javaexec the receiver, thus necessitating `this@javaexec`. It's not very pretty - adjusting the arg name would help